api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -379,6 +381,11 @@ func (c *imageBuilder) createKanikoJobSpec(project mlp.Project, model *models.Mo
 			TTLSecondsAfterFinished: &jobTTLSecondAfterComplete,
 			ActiveDeadlineSeconds:   &activeDeadlineSeconds,
 			Template: v1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      kanikoPodName,
```

Comment:
I don't think you can specify pod name and namespace since it'll be controlled by the kube cluster. Can you check whether it works that way?